logbook-httpclient/src/main/java/org/zalando/logbook/httpclient/LogbookHttpRequestInterceptor.java

```diff
@@ -22,9 +24,13 @@ public LogbookHttpRequestInterceptor(final Logbook logbook) {
 
     @Override
     public void process(final HttpRequest httpRequest, final HttpContext context) throws IOException {
-        final LocalRequest request = new LocalRequest(httpRequest);
-        final ResponseProcessingStage stage = logbook.process(request).write();
-        context.setAttribute(Attributes.STAGE, stage);
+        try {
+            final LocalRequest request = new LocalRequest(httpRequest);
+            final ResponseProcessingStage stage = logbook.process(request).write();
+            context.setAttribute(Attributes.STAGE, stage);
+        } catch (Exception e) {
+            log.trace("Unable to log request: {}", e.getClass());
```

Comment:
See my comment above on log level and wording. Also: why log the exception class explicitly (instead of passing the exception as the last argument)?

Reply:
> why log the exception class explicitly (instead of passing the exception as the last argument)? 

Can't remember why I did this to be honest. Changed to passing the whole exception as in other places